newIDE/app/src/UI/EditorMosaic/style.css

```diff
@@ -23,7 +23,7 @@
 
 /* Make the horizontal split to have the same thickness as the vertical */
 .mosaic-gd-theme .mosaic-split.-column {
-  margin-top: -2px;
+  margin-top: -3px;
```

Comment:
hmm, this feels like a random change. Are we sure it doesn't affect anything else?

Reply:
It's not random, it displays enough for it to be the same size as for the vertical handles